launch/launch/event_handler.py

```diff
@@ -19,13 +19,13 @@
 from typing import Optional
 from typing import Text
 from typing import Tuple
+from typing import TYPE_CHECKING
 
 from .event import Event
 from .some_actions_type import SomeActionsType
 
-if False:
-    # imports here would cause loops, but are only used as forward-references for type-checking
-    from .launch_context import LaunchContext  # noqa
+if TYPE_CHECKING:
+    from .launch_context import LaunchContext
```

Comment:
Don't delete the `# noqa` comment.
I think it was causing problems in some ci jobs. See https://github.com/ros2/launch/pull/274.

And maybe, `#noqa: F401` is better.

Reply:
same in other places